Makefile

```diff
@@ -36,11 +36,13 @@ haddock-shallow:
 # formats all Haskell files (which don't contain CPP)
 .PHONY: format
 format:
+	licensure -p
 	./tools/ormolu.sh
 
 # formats all Haskell files even if local changes are not committed to git
 .PHONY: formatf
 formatf:
+	licensure -p
```

Comment:
If we do this, we should also add a CI check that running licensure is idempotent.

Reply:
It actually isn't idempotent, see my review.